common/src/primitives.rs

```diff
@@ -200,6 +200,8 @@ mod _allowed_deprecated {
         KGOLD = 13,
         KXOR = 14,
         KARMA = 15,
+        #[cfg(feature = "wip")] // presto
```

Comment:
I think it's better to not use feature flag here, because it will make troubles with type generation on frontend side. We can just "reserve" this asset here without registration and when we're ready we will register it

Reply:
There is a chance that the feature will be declined by WB